lib/capistrano/application.rb

```diff
@@ -63,13 +63,10 @@ def top_level_tasks
 
     def display_error_message(ex)
       unless options.backtrace
-        if (loc = Rake.application.find_rakefile_location)
-          whitelist = (@imported.dup << loc[0]).map{|f| File.absolute_path(f, loc[1])}
-          pattern = %r@^(?!#{whitelist.map{|p| Regexp.quote(p)}.join('|')})@
-          Rake.application.options.suppress_backtrace_pattern = pattern
-        end
-        trace "(Backtrace restricted to imported tasks)"
+        Rake.application.options.suppress_backtrace_pattern = backtrace_pattern
```

Comment:
This has the effect of setting `Rake.application.options.suppress_backtrace_pattern` to `nil` if `loc` is falsey, which wasn't the case before the refactor.